docs/multiple_updates_for_same_buffer.md

```diff
@@ -0,0 +1,76 @@
+This is copied from #197
+
+### Multiple updates for a SSBO in VkRunner
+
+As debated [here](https://github.com/Igalia/vkrunner/issues/46),
+VkRunner applies multiple updates for a SSBO and uses the final
+one for all computes or draws.
+
+For example,
+```
+ssbo 0 subdata float 0 1.0
+compute 1 1 1
+ssbo 0 subdata float 0 2.0
+compute 1 1 1
+ssbo 0 subdata float 36 1.0 2.0 3.0 \
+                        ... ( more than 4000 numbers) ... 4096.0
+compute 1 1 1
+probe ssbo float 0 ~= 2.0
+ssbo 0 subdata float 4000 1.0 2.0 3.0 \
+                          ... ( more than 4000 numbers) ... 4096.0
+compute 1 1 1
+probe ssbo float 4000 ~= 1.0
+```
+
+This script has the same behavior as the following script does
```

Comment:
That's not quite correct. It uses all of them up to the probe. The probe commands submit the command buffer which collapses all the ssbo buffers. So, the first 3 ssbo commands will be used with the first 3 compute calls. The 4th ssbo command will be used with the 4th compute command.